.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+Doc/library/hashlib.rst        @tiran
```

Comment:
Could this be collapsed with `Lib/hashlib.py` with `hashlib.*`? Same for `ssl.*`? And if you really want to catch everything including tests with one rule then would `*hashlib.*` work?